scripts/android/before_install.sh

```diff
@@ -19,5 +19,5 @@ docker run -v $PWD:/android/subsurface \
 	   -v $PARENT/subsurface-mobile-build-docker-arm64:/android/subsurface-mobile-build-arm64 \
 	   --name=android-builder \
 	   -w /android \
-	   -d dirkhh/android-builder:5.12.04.1 \
+	   -d subsurface/android-build-container:5.13.10 \
```

Comment:
we no longer use Travis, so this is basically dead code...

Reply:
see below.